src/main/java/org/jenkinsci/main/modules/cli/auth/ssh/UserPropertyImpl.java

```diff
@@ -55,6 +58,30 @@ public String getDisplayName() {
         public UserProperty newInstance(User user) {
             return null;
         }
+
+        @SuppressWarnings("unused")
+        public FormValidation doCheckAuthorizedKeys(@QueryParameter String authorizedKeys, @AncestorInPath User currentUser) {
+            for (User user: User.getAll()) {
+                UserPropertyImpl userPropertyImpl = user.getProperty(UserPropertyImpl.class);
+                if (!authorizedKeys.isEmpty() && userPropertyImpl!=null && userPropertyImpl.authorizedKeys!=null && user!=currentUser && checkReapeatedAuthorizedKeys(authorizedKeys, userPropertyImpl)) {
```

Comment:
There is no reason to loop through the users in case `authorizedKeys.isEmpty()`.